MoltenVK/MoltenVK/GPUObjects/MVKPipeline.mm

```diff
@@ -25,9 +25,11 @@
 #include "MTLRenderPipelineDescriptor+MoltenVK.h"
 #include "mvk_datatypes.hpp"
 
+#ifndef MVK_EXCLUDE_CEREAL
```

Comment:
Please modify this to align with the conventions for definitions of other `MVK_` build switches. Something like:

```
#ifndef MVK_USE_CEREAL
#	define MVK_USE_CEREAL	1
#endif
```

Then when using:

```
#if MVK_USE_CEREAL
...
[#else
...]
#endif
```

Reply:
Ok, done.